### DISCUSSION TITLE
Klayout-integration installation error. The new version cannot be displayed in real time on Klayout.

### DISCUSSION THREAD

**Discussion by LuYouxi** [gdsfactory/gdsfactory#1711] [Q&A] [answered]
Cannot install klayout intergration when installing new version gdsfactory. The error is as shown in the figure below:
![image](https://github.com/gdsfactory/gdsfactory/assets/35725059/4b5a8084-1f50-4721-9c66-24ffa501879d)
My operating system version is as follows:
![image](https://github.com/gdsfactory/gdsfactory/assets/35725059/0fe64e0a-f5cd-464c-bb03-2a5c27aa5ca1)

**Comment by joamatab** [comment, marked as answer]
Hi, 

here is the fix
https://github.com/gdsfactory/gdsfactory/pull/1712


can you try again?

`pip install gdsfactory --upgrade`

and then 
```
gf install klayout-genericpdk
```

**Reply by LuYouxi** [reply]
Thank you. That's fine now.
